### PR TITLE
Doctest fix

### PR DESCRIPTION
**Pull request by ghackebeil**
## Fixes # .

Fixes execution of doctests

## Summary/Motivation:

 - `make doctests` was failing due to unknown target `doctests` (just needed to remove "s").
 - failure return code was being overwritten by a later command (replaced ; with &&)
 - needed to `pip install sphinx`

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the BSD license.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -163,7 +163,7 @@ script:
   - test.pyomo -v --cat=$CATEGORY pyomo `pwd`/pyomo-model-libraries
 
   # Run documentation tests
-  - cd doc/OnlineDocs; make doctests -d; cd ../..
+  - cd doc/OnlineDocs && make doctest -d && cd ../..
```
Comment (jsiirola) [inline .travis.yml]: This worries me.  If doctest fails, then the CWD is not restored back to outside of the doc directory.  I have frequently used:
```
pushd doc/OnlineDocs
make doctest -d # possible also '|| exit 1'
popd
```
when I was working with scripts that had to change/restore the CWD.

**Comment by ghackebeil** [comment]
maybe just just delete the `&& cd ..` and add another line in the travis file immediately after with something like `cd $TRAVIS_BUILD_DIR`

**Comment by ghackebeil** [comment]
Also, if doctests fails, would travis stop executing commands anyway? If so, what's the problem.

**Comment by jsiirola** [comment]
I am not sure if Travis aborts on nonzero return codes.  For example, nose / test.pyomo return nonzero return codes any time a test fails, yet the build continues...

**Comment by ghackebeil** [comment]
Ah good point. What about just using `make -C doc/OnlineDocs …`?

**Pull request merged**